src/blocktypes/actions/run.nix

```diff
@@ -0,0 +1,18 @@
+{
+  lib,
+  target,
+}: let
+  name = lib.removeSuffix "-${target.version or ""}" target.name;
```

Comment:
Please use the contract accessor, here.

https://github.com/nix-community/nixpkgs.lib/blob/master/lib/strings.nix#L601-L616

Reply:
fixed in https://github.com/divnix/std/pull/206/commits/151d225b948b257b71cb2aaee6fcd84f491f2671